torch/_inductor/scheduler.py

```diff
@@ -661,7 +661,7 @@ def get_estimated_runtime(self) -> float:
             assert isinstance(self.node, ir.IRNode)
             try:
                 return estimate_nccl_collective_runtime(self.node)
-            except ValueError as e:
+            except Exception as e:
```

Comment:
Curious: what are the other exception types that we saw?

Reply:
It is a `TypeError`, traceback is 
```
   File ".../torch/_inductor/scheduler.py", line 653, in get_estimated_runtime
     return estimate_nccl_collective_runtime(self.node)
   File ".../torch/_inductor/comm_analysis.py", line 181, in estimate_nccl_collective_runtime
     group_size = get_collective_group_size(node)
   File ".../torch/_inductor/comm_analysis.py", line 75, in get_collective_group_size
     raise TypeError(f"Unsupported collective type: {node}")
```